experimental/framework/frs-client/src/FrsAudience.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ServiceAudience } from "@fluid-experimental/fluid-static";
+import { IClient } from "@fluidframework/protocol-definitions";
+import { IFrsAudience, FrsMember } from "./interfaces";
+
+export class FrsAudience extends ServiceAudience implements IFrsAudience {
```

Comment:
I suspect we'll eventually want to compose rather than extend here, but I think this is OK for now.

Reply:
This might be for a separate PR, as this is currently just matching the format we have set for generating the audience for both `TinyliciousClient` and `OdspClient`. Each service can then extend their own audience details as they see fit. What would we be locking ourselves into here?

Reply:
The change would be to drop the concrete base class (the `IFrsAudience` interface probably doesn't need to change for that to happen)

Reply:
So I don't think there's any lock in to worry about if we do it later.